tests/integration/s2n_handshake_test_s_client.py

```diff
@@ -272,10 +272,19 @@ def try_handshake(endpoint, port, cipher, ssl_version, server_name=None, strict_
     if use_corked_io:
         s2nd_cmd.append("-C")
 
-    s2nd = subprocess.Popen(s2nd_cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE)
+    s2nd = subprocess.Popen(s2nd_cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
 
     # Make sure s2nd has started
-    s2nd.stdout.readline()
+    print("Waiting for s2nd to start")
+    try:
+        out, err = s2nd.communicate(timeout=2)
+        print("out: {}".format(out))
+        print("err: {}".format(err))
+    except subprocess.TimeoutExpired as e:
+        pass
+
+    #s2nd.stdout.readline()
```

Comment:
artifact?

Reply:
junk PR to test travis. I set up Travis in my personal repo though so I don't have to clutter the main repo any more.